apps/crn-server/test/controllers/research-outputs.test.ts

```diff
@@ -1038,7 +1044,9 @@ describe('ResearchOutputs controller', () => {
         expect(squidexGraphqlClientMock.request).toHaveBeenCalledWith(
           expect.anything(),
           expect.objectContaining({
-            filter: `(data/link/iv eq '${researchOutputRequest.link}')`,
+            filter: `(data/link/iv eq '${sanitiseForSquidex(
+              researchOutputRequest.link || '',
+            )}')`,
```

Comment:
I don't think we should be asserting using sanitiseForSquidex. It's from the production code, and it's part of the method under test.